### ISSUE TITLE
Choices don't follow examples

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8401]
The examples at the top all have mm, cm, in, attached,
![20210314T142622](https://user-images.githubusercontent.com/1230959/111059546-6441d780-84d1-11eb-99e7-7fd99fc40cf0.jpg)
But the choices below all lack them!

Seen on
`man_made=pipeline
`

**Comment by mxdanger** [comment]
You may want to either close or rename your issue. The only issue here is that the example includes the highly discouraged alias of `in`, which should be `"`.

The list of suggestions all valid with the exception of the negative integer value. Unless otherwise specified, the unit of measure for `diameter=*` is mm, so the absence of any unit prefix means that the value is in millimeters. This is the same thing for `width=*` except the default is meters.

I suggest you click the "i" button to get information about the tag before you assume the suggested choices are either correct or incorrect.

**Comment by jidanni** [comment]
So the first line should be:
`5, 10 cm, 15 "`
perhaps. Or even clearer:
`5, 5 mm, 10 cm, 15 "`
P.S., `15"` would seem more customary than `15 "`.

**Comment by mxdanger** [comment]
> P.S., `15"` would seem more customary than `15 "`.

Yes, `15"` is the intended unit prefix. See https://wiki.openstreetmap.org/wiki/Units for more information.
